metricq/history_client.py

```diff
@@ -526,8 +537,7 @@ async def history_last_value(self, metric: str, timeout=60) -> Optional[TimeValu
                 Operation timeout in seconds.
 
         Raises:
-            InvalidHistoryResponse:
-                if an invalid response was received
+            ~exceptions.InvalidHistoryResponse:
```

Comment:
It seems redundant, but every raises had a short description for when the exception is raised. I'd add it for consistency